### PR TITLE
[IR-2057] Unit Tests: Hierarchy `EntityTree.tsx`

### PR DESCRIPTION
**Pull request by heysokam**
## Summary
- [x] `useChildWithComponent`
- [x] `useAncestorWithComponent`

### CODE REVIEW

File: packages/spatial/src/transform/components/EntityTree.test.tsx [hunk 1]
```diff
@@ -44,9 +46,32 @@ import {
   removeFromEntityTree,
   traverseEntityNode,
   traverseEntityNodeParent,
+  useAncestorWithComponent,
+  useChildWithComponent,
   useTreeQuery
 } from './EntityTree'
 
+const Validate = {
```
Comment (HexaField) [inline packages/spatial/src/transform/components/EntityTree.test.tsx]: let's just make this a function `assertEntityTree`

**Review by HexaField — DISMISSED** [review]

File: packages/spatial/src/transform/components/EntityTree.test.tsx (hunk 1 above)
Comment (heysokam) [reply packages/spatial/src/transform/components/EntityTree.test.tsx]: Changed with 1fce025

**Comment by DanielBelmes** [comment]
@heysokam looks like tests are failing

**Comment by heysokam** [comment]
> @heysokam looks like tests are failing

@DanielBelmes I believe the branch was on an older dev where tests where failing
I just merged latest dev and it seems to be passing now :writing_hand:

**Review by DanielBelmes — APPROVED** [review]

**Comment by heysokam** [comment]
@DanielBelmes Should we merge it, or is something else needed?

**Pull request merged**